content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -436,192 +618,274 @@ Resources are deleted in two phases: 1) finalization, and 2) removal.
 }
 ```
 
-When a client first deletes a resource, the `.metadata.deletionTimestamp` is set to the current time.
+When a client first **delete**s a resource, the `.metadata.deletionTimestamp` is set to the current time.
 Once the `.metadata.deletionTimestamp` is set, external controllers that act on finalizers
 may start performing their cleanup work at any time, in any order.
-Order is NOT enforced because it introduces significant risk of stuck `.metadata.finalizers`.
-`.metadata.finalizers` is a shared field, any actor with permission can reorder it.
-If the finalizer list is processed in order, then this can lead to a situation
+
+Order is **not** enforced between finalizers because it would introduce significant
+risk of stuck `.metadata.finalizers`.
+
+The `.metadata.finalizers` field is shared: any actor with permission can reorder it.
+If the finalizer list were processed in order, then this might lead to a situation
 in which the component responsible for the first finalizer in the list is
-waiting for a signal (field value, external system, or other) produced by a
+waiting for some signal (field value, external system, or other) produced by a
 component responsible for a finalizer later in the list, resulting in a deadlock.
-Without enforced ordering finalizers are free to order amongst themselves and
-are not vulnerable to ordering changes in the list.
+
+Without enforced ordering, finalizers are free to order amongst themselves and are
+not vulnerable to ordering changes in the list.
 
 Once the last finalizer is removed, the resource is actually removed from etcd.
 
 
 ## Single resource API
 
-API verbs GET, CREATE, UPDATE, PATCH, DELETE and PROXY support single resources only.
-These verbs with single resource support have no support for submitting
-multiple resources together in an ordered or unordered list or transaction.
-Clients including kubectl will parse a list of resources and make
-single-resource API requests.
+The Kubernetes API verbs **get**, **create**, **update**, **patch**, **delete** and
```

Comment:
The original text used the HTTP verbs. Is this change correct?

Reply:
There is no CREATE verb in HTTP; the original text was perhaps misleading.

Reply:
Right, you would use `POST`.

Reply:
nit: This section seems out of place, mostly because the previous sections describe collections.
Possibly move after, https://deploy-preview-25299--kubernetes-io-main-staging.netlify.app/docs/reference/using-api/api-concepts/#resource-uris

Reply:
If listing the k8s verbs, this should include APPLY